.github/README.md

```diff
@@ -34,14 +34,25 @@ If you shutdown Docker and want to fire up the environment later, use `yarn env:
 
 ### Enable X-Debug
 
-To enable X-Debug in the containers, you need to set the environmental variables before starting up Docker:
-
+To enable X-Debug in the containers. You need to set the environmental variables before starting up Docker:
 * `export LOCAL_PHP_XDEBUG=true`
 * `export XDEBUG_HOST_IP=10.0.0.1`
+NOTE: change the IP `10.0.0.1` to the value of`$_SERVER[‘REMOTE_ADDR’]` when you load up the Docker-powered site.
+
+For Ubuntu users, run the command `ip a` and find the `docker0` alias. Copy the inet value.
```

Comment:
We should make it clear that Linux users (not just Ubuntu users) should not use the `$_SERVER['REMOTE_ADDR']`. Instead, they need to get the IP address assigned to the `docker0` network interface.